README.md

```diff
@@ -11,11 +11,11 @@ Due to security, when installed this way the Magento standard for phpcs cannot b
 You can achieve this by adding the following to your project's `composer.json`:
 ````
 "scripts": {
-    "post-install-cmd": [
-      "([ $COMPOSER_DEV_MODE -eq 0 ] || vendor/bin/phpcs --config-set installed_paths ../../magento/magento-coding-standard/)"
+"post-install-cmd": [
+      "( [ -f vendor/bin/phpcs ] && [ -L vendor/bin/phpcs ] && vendor/bin/phpcs --config-set installed_paths ../../magento/magento-coding-standard/) || true"
```

Comment:
Could you please investigate the possibility to avoid the use of `|| true` here?